.github/workflows/vast.yaml

```diff
@@ -942,20 +949,34 @@ jobs:
           fail-if-no-release: true
           assets: "${{ matrix.nix.target }}-linux-static.tar.gz"
 
-      - name: Upload Release Assets
+      - name: Upload Release Tarball
         if: github.event_name == 'release'
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ github.event.release.upload_url }}
-          asset_path: "${{ env.BUILD_DIR }}/${{ steps.create_paths.outputs.artifact_name }}"
+          asset_path: "${{ steps.build_static_package.outputs.store_path }}/${{ steps.get_artifact_names.outputs.tar_gz }}"
           # The asset name is constant so we can permanently link to
           # https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.tar.gz
           # for a build of the latest release.
           asset_name: "${{ matrix.nix.target }}-linux-static.tar.gz"
           asset_content_type: application/gzip
 
+      - name: Upload Debian Release Package
+        if: github.event_name == 'release'
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ github.event.release.upload_url }}
+          asset_path: "${{ steps.build_static_package.outputs.store_path }}/${{ steps.get_artifact_names.outputs.deb }}"
+          # The asset name is constant so we can permanently link to
+          # https://github.com/tenzir/vast/releases/latest/download/vast-linux-static.deb
```

Comment:
We should update the docs on `web/docs/setup-vast/download.md` to reference the Deb in addition to the tar.gz.